azure_functions_worker/logging.py

```diff
@@ -76,4 +76,4 @@ def enable_console_logging() -> None:
 def is_system_log_category(ctg: str) -> bool:
     # Category starts with 'azure_functions_worker' or
     # 'azure_functions_worker_errors' will be treated as system logs
-    return ctg.lower().startswith('azure_functions_worker')
+    return ctg.lower().startswith(SYSTEM_LOG_PREFIX)
```

Comment:
Same question - `SYSTEM_LOG_PREFIX`, `CONSOLE_LOG_PREFIX` can be moved here. Better have the variables where they are used.